src/Microsoft.OData.Client/Build.Portable/Microsoft.OData.Client.Portable.cs

```diff
@@ -175,6 +175,7 @@ internal sealed class TextRes {
         internal const string ALinq_CantAddDuplicateQueryOption = "ALinq_CantAddDuplicateQueryOption";
         internal const string ALinq_CantAddAstoriaQueryOption = "ALinq_CantAddAstoriaQueryOption";
         internal const string ALinq_CantAddQueryOptionStartingWithDollarSign = "ALinq_CantAddQueryOptionStartingWithDollarSign";

```

Comment:
After you replaced this in the ResourceBinder, is it still used anywhere else other than in the test?

Reply:
Didn't see it use anywhere else, I've removed it.